libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -17,12 +17,12 @@ export const addToReadingList = createAction(
   props<{ book: Book }>()
 );
 
-export const failedAddToReadingList = createAction(
+export const addToReadingListFailed = createAction(
```

Comment:
I agree with this change though - I do tend to prefer the past tense verb as a suffix. 👍